apps/src/sites/hourofcode.com/pages/views/analytics_event_log_helper.js

```diff
@@ -0,0 +1,10 @@
+import $ from 'jquery';
+
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+
+// Log page visits on document.ready.
+$(document).ready(() => {
+  const eventLogger = document.getElementById('analytics-event-log-helper');
+  const pageVisitEventName = eventLogger?.dataset?.eventName;
+  analyticsReporter.sendEvent(pageVisitEventName);
```

Comment:
@kelbyhawn @hannahbergam should this only be logging to statsig?

Reply:
Yep! Updated this here: https://github.com/code-dot-org/code-dot-org/pull/60838